github/PullRequest.py

```diff
@@ -491,7 +491,6 @@ def create_review(
         assert is_optional(event, str), event
         assert is_optional_list(comments, dict), comments
         post_parameters: dict[str, Any] = NotSet.remove_unset_items({"body": body})
-        post_parameters["event"] = "COMMENT" if is_undefined(event) else event
```

Comment:
This will remove the `event` argument all together.

Reply:
Gotcha, so by adding it to `NotSet.remove_unset_items` it'll handle not passing `event` for us?

Reply:
That is, provided the caller doesn't use the `event` argument

Reply:
Yes, if `event` is not given by caller, `NotSet.remove_unset_items` will remove it from the dict.